app/controllers/events_controller.rb

```diff
@@ -5,6 +5,8 @@ class EventsController < ApplicationController
   before_action :authorize_event_author, only: [:update, :destroy]
 
   def index
+    @events = @project.events
+    @project_id = @project.id
```

Comment:
I need to pass project or project id to the link helper in the view. I don't want to mix concerns so I don't use params in view, this way I can easily test my view in separation from controller.